# 请求时发送的数字
int32 num1
int32 num2
---
# 相应时发送的数字
int32 sum